src/styles/blocks/_stopwatch.scss

```diff
@@ -0,0 +1,55 @@
+.stopwatch {
+  position: absolute;
+  width: 500px;
+  height: 500px;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  border: 10px solid rgb(127, 255, 212);
+  background-color: #fff;
+  border-radius: 50%;
+  margin: auto;
+  box-shadow: 20px 20px 10px 15px rgba(0, 0, 255, 0.2);
+}
+
+.stopwatch__center {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  width: 50px;
+  height: 50px;
+  margin: auto;
+  border-radius: 50%;
+  background-color: rgb(0, 128, 128);
+}
+
+.stopwatch__second {
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin: auto;
+  top: 50px;
+  width: 15px;
+  height: 200px;
+  background-color: #4c9aaa;
+  border-radius: 60px;
+  animation: move 60s steps(60, end) infinite;
+  transform-origin: bottom;
+}
+
+.stopwatch__minute {
```

Comment:
make sure to fix the styles of this element the same way as I've mentioned in the previous comments